src/main/java/net/sf/marineapi/nmea/sentence/SentenceId.java

```diff
@@ -41,6 +41,9 @@
     DBT,
     /** Depth of water below transducer; in meters. */
     DPT,
+	/** Boreal GasFinder data - added by Bob Schwarz*/
+	DTA,
+	DTB,
```

Comment:
Indentations, in some other files too.

Reply:
All tabs replaced with spaces.  Alignment looks good in Eclipse.